elfi/methods/inference/samplers.py

```diff
@@ -374,19 +375,29 @@ def set_objective(self, n_samples, thresholds=None, max_iter=10,
             Bayesian Analysis 1:1-27, 2021. https://doi.org/10.1214/20-BA1211
 
         """
-        self.adaptive_quantile = initial_quantile
-        self.adaptive_threshold_value = np.Inf
-
+        # 1: Adaptive threshold or predetermined thresholds
```

Comment:
Numbers from comments should be removed, as they are read vertically anyways starting from the top.

Reply:
ok!